integration-tests/smoke/vrfv2_test.go

```diff
@@ -1158,6 +1155,7 @@ func TestVRFV2NodeReorg(t *testing.T) {
 			*configCopy.VRFv2.General.NumberOfWords,
 			*configCopy.VRFv2.General.RandomnessRequestCountPerRequest,
 			*configCopy.VRFv2.General.RandomnessRequestCountPerRequestDeviation,
+			0,
```

Comment:
nit: you could define `const ROOT_KEY_INDEX = 0` and use that instead of `0, so that it's clear what it means

Reply:
@Tofel do we have this const in Seth maybe?